Makefile

```diff
@@ -120,33 +120,56 @@ start:
 	CONTIV_DOCKER_VERSION="$${CONTIV_DOCKER_VERSION:-$(DEFAULT_DOCKER_VERSION)}" CONTIV_NODE_OS=${CONTIV_NODE_OS} vagrant up
 endif
 
+# ===================================================================
 #kubernetes demo targets
-k8s-cluster:
-	cd vagrant/k8s/ && CONTIV_K8S_USE_KUBEADM=1 ./setup_cluster.sh
-k8s-legacy-cluster:
-	cd vagrant/k8s/ && ./setup_cluster.sh
-k8s-l3-cluster:
-	CONTIV_L3=1 make k8s-cluster
 k8s-demo:
 	cd vagrant/k8s/ && ./copy_demo.sh
+
 k8s-demo-start:
 	cd vagrant/k8s/ && ./restart_cluster.sh && vagrant ssh k8master
+
+# ===================================================================
```

Comment:
I've included the comments as well (next line). The lines are just demarcating the targets that are relevant for the comments.